settings.gradle.kts

```diff
@@ -1,5 +1,14 @@
 rootProject.name = "avito-android-infra"
 
+// Convenience to provide constants for library GA coordinates
+includeBuild("libraries")
+
+// Platform for dependency versions shared by 'main build' and 'build-logic'
+includeBuild("platform")
+
+// Not traditional 'buildSrc', but 'build-logic' as a normal included build
+includeBuild("build-logic")
+
```

Comment:
How about grouping them into one folder?
As for me, these are implementation details and don't need at a root level.
E.g. `libraries` sounds like regular modules.

Reply:
you mean all inside `build-logic`?
i'd prefer to merge it like this, test if all working correctly and then move it as we feel right, because it's not in a public api

Reply:
Ok